### PR TITLE
Feature/e UI 3223 integrate spinner aa

### PR DESCRIPTION
**Pull request by adnanakgun**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-3223

### Change description ###

Integrated spinner service from both libraries. Also the spinner component from common lib.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by phillip-whitaker — APPROVED** [review]
lgtm

File: src/cases/containers/case-home/case-home.component.ts [hunk 1]
```diff
@@ -25,11 +24,15 @@ export class CaseHomeComponent implements OnInit, OnDestroy {
...
   public navigationSubscription: Subscription;
 
+  public showSpinner$: Observable<boolean>;
+
   constructor(
     private readonly alertService: AlertService,
     private readonly errorNotifierService: ErrorNotifierService,
     private readonly navigationNotifier: NavigationNotifierService,
     private readonly store: Store<fromFeature.State>,
+    private commonLibLoadingService: CommonLibLoadingService,
```
Comment (paul-graham) [inline src/cases/containers/case-home/case-home.component.ts:34]: I'll approve the merge for now but can these be changed to `readonly`, please?

File: src/cases/containers/case-home/case-home.component.ts [hunk 2]
```diff
@@ -45,6 +48,13 @@ export class CaseHomeComponent implements OnInit, OnDestroy {
         this.actionDispatcher(this.paramHandler(navigation));
       }
     }) as any;
+
+    this.store.dispatch(new fromRoot.StartIdleSessionTimeout());
```
[the remark this answers is not in the document]
Comment (adnanakgun) [reply src/cases/containers/case-home/case-home.component.ts]: I don't know how that snuck in. Wasnme... Removed it now.

File: src/cases/containers/case-home/case-home.component.ts (hunk 1 above)
Comment (adnanakgun) [reply src/cases/containers/case-home/case-home.component.ts]: Done

**Review by paul-graham — APPROVED** [review]
All looking good. Thanks for putting my mind at rest by making those changes, @adnanakgun. 😁

**Pull request merged**